daemon/src/model/cfd.rs

```diff
@@ -268,6 +268,18 @@ pub enum RolloverError {
     },
 }
 
+/// Errors that can happen when handling the expiry of the refund
+/// timelock on the commit transaciton.
+#[derive(thiserror::Error, Debug)]
+pub enum RefundTimelockExpiryError {
+    #[error("CFD is already final")]
+    AlreadyFinal,
+    #[error("CFD does not have a DLC")]
+    NoDlc,
+    #[error("Failed to sign refund transaction: {0}")]
```

Comment:
```suggestion
    #[error("Failed to sign refund transaction")]
```

Otherwise you are duplicating the error message.

Reply:
Don't I need to annotate it as `source` then? Keep in mind that we are not using `anyhow::Context` where this error can occur.

Reply:
`#[from]` implies `#[source]`: https://docs.rs/thiserror/latest/thiserror/index.html#details